.github/workflows/rw-versions-smoke-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Reusable workflow to run smoke tests on different k8s versions
```

Comment:
nit - Instead of using `rw` as prefix, can we move them to `reusable/` or `templates/`? It took me a while to figure out what it meant.

Reply:
🤦 resolving

Reply:
Can we then at least rename it from `rw` to `template`?

Reply:
sure, something like `template-XXXXX`?

Reply:
Yes, please ❤️

Reply:
done :)